site/gatsby-site/playwright/e2e-full/pageCreators/createReportPages.spec.ts

```diff
@@ -0,0 +1,118 @@
+import sinon from 'sinon';
+import createReportPages from '../../../page-creators/createReportPages';
+import { CreatePagesArgs } from 'gatsby';
+import { expect } from '@playwright/test';
+import { test } from '../../utils';
+
+test.describe('createReportPages', () => {
+  let graphql: sinon.SinonStub;
+  let createPage: sinon.SinonSpy;
+
+  const response = {
+    data: {
+      reports: {
+        nodes: [
+          { report_number: 1, language: 'en' },
+          { report_number: 2, language: 'es' },
+        ],
+      },
+    },
+  };
+
+  const languages = [
+    { code: 'en', hrefLang: 'en-US', name: 'English', localName: 'English', langDir: 'ltr', dateFormat: 'MM/DD/YYYY' },
+    { code: 'es', hrefLang: 'es', name: 'Spanish', localName: 'Español', langDir: 'ltr', dateFormat: 'DD-MM-YYYY' },
+    { code: 'fr', hrefLang: 'fr', name: 'French', localName: 'Français', langDir: 'ltr', dateFormat: 'DD-MM-YYYY' },
+    { code: 'ja', hrefLang: 'ja', name: 'Japanese', localName: '日本語', langDir: 'ltr', dateFormat: 'YYYY/MM/DD' },
+  ];
```

Comment:
Please refactor this collection of languages by extracting the definition into a common file for reuse